Test/PcapDeviceTest.cs

```diff
@@ -52,9 +62,65 @@ public void DeviceProperties([PcapDevices] DeviceFixture fixture)
             Assert.IsNotNull(pcapIf.GatewayAddresses);
             Assert.IsNotNull(pcapIf.Addresses);
 
-            if (pcapIf.MacAddress != null)
+            try
+            {
+                var resolution = device.TimestampResolution;
+                if (targetResolution != null)
+                {
+                    // confirm the resolution was set
+                    if (targetResolution != resolution)
+                    {
+                        Console.WriteLine("Error: {0} expected {1} but was {2}", device.Name, targetResolution, resolution);
+                    }
+                    else
+                    {
+                        Console.WriteLine("Pass: {0} expected {1} and was {2}", device.Name, targetResolution, resolution);
+                    }
+
+                    // Note: This doesn't not appear to work for any platform so it is
+                    // disabled for now
+                    //                Assert.AreEqual(targetResolution, resolution);
+                }
+            } catch (System.EntryPointNotFoundException)
```

Comment:
`EntryPointNotFoundException` should not propagte outside of the library, it should be wrapped into NotSupportedException